tests/cli/test_ci.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+set -e
+
+. /usr/share/beakerlib/beakerlib.sh
+
+# I guess that all of this could be moved to setup phase
+
+cat >> /etc/yum.repos.d/nightly.repo << __EOF__
+
+[RHEL-8-NIGHTLY-CRB]
+name=CRB
+baseurl=http://download-ipv4.eng.brq.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.2/compose/CRB/x86_64/os/
```

Comment:
Seems like a fragile URL, and possibly not local to wherever the test runs. Is there a more general one that can be used?

Reply:
Thanks for noticing, I guess I got redirected when I was looking for it. I changed it to the one you proposed.